docs/json-rpc-endpoints.md

```diff
@@ -0,0 +1,115 @@
+# JSON RPC Endpoints
+
+## ETH
+
+### Supported
+
+- eth_blockNumber
+- eth_chainId
+- eth_gasPrice
+- eth_getBlockByHash
+- eth_getBlockByNumber
+- eth_getFilterChanges
+- eth_getFilterLogs
+- eth_getLogs
+- eth_getTransactionByBlockHashAndIndex
+- eth_getTransactionByHash
+- eth_getTransactionCount
+- eth_getBlockTransactionCountByHash
+- eth_getBlockTransactionCountByNumber
+- eth_getTransactionReceipt
+- eth_newBlockFilter
+- eth_newFilter
+- eth_uninstallFilter
+- eth_syncing
+- eth_subscribe
+- eth_unsubscribe
+
+### Differences
+
+- eth_call _* doesn't support state override at the moment and pending block_ 
+- eth_estimateGas _* if the block number is set to pending we assume it is the latest_
+- eth_getBalance _* if the block number is set to pending we assume it is the latest_
+- eth_getCode _* if the block number is set to pending we assume it is the latest_
+- eth_getCompilers _* response is always empty_
+- eth_getStorageAt _* if the block number is set to pending we assume it is the latest_
+- eth_getTransactionByBlockNumberAndIndex _* if the block number is set to pending we assume it is the latest_
+- eth_newPendingTransactionFilter _* not supported yet_
+- eth_sendRawTransaction _* can relay TXs to another node_
```

Comment:
I wouldn't consider this a discrepancy, as geth nodes do the same through P2P. If anything the difference could be that the tx is not added atomically to the pool (needs to come and go to the trusted sequencer). Actually maybe it would be a good idea to add txs to the pool before being relayed for this reason

Reply:
I just added this as a note, so everyone can be aware of it. Should we remove it?

Reply:
it's ok